frontend/src/metabase-lib/mocks.ts

```diff
@@ -15,12 +15,11 @@ import {
 import Question from "metabase-lib/Question";
 import NativeQuery from "metabase-lib/queries/NativeQuery";
 import StructuredQuery from "metabase-lib/queries/StructuredQuery";
-import Query from "metabase-lib/queries/Query";
 
 export type NativeSavedCard = SavedCard<NativeDatasetQuery>;
-export type NativeUnsavedCard = UnsavedCard<NativeDatasetQuery>;
+type NativeUnsavedCard = UnsavedCard<NativeDatasetQuery>;
```

Comment:
This is used in https://github.com/metabase/metabase/blob/45f77dbb5edae64bf09bfc5683cd6e74d18178d9/frontend/src/metabase/models/containers/ModelDetailPage/ModelDetailPage.unit.spec.tsx#L146, so it should still be exported.